cinema/models.py

```diff
@@ -84,23 +88,35 @@ class Ticket(models.Model):
     row = models.IntegerField()
     seat = models.IntegerField()
 
+    @staticmethod
+    def validate_seats(
+            seat: int,
+            seat_in_row: int,
+            row: int,
+            rows: int,
+            error_to_raise
```

Comment:
You could indicate that `error_to_raise` should be an exception in the type annotation.